docs/self-managed/identity/user-guide/tenants/managing-tenants.md

```diff
@@ -0,0 +1,84 @@
+---
+id: managing-tenants
+title: "Managing tenants"
+sidebar_label: "Managing tenants"
+description: "Learn about the methods to manage tenants within the Identity application."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+In this guide you will learn about how to manage tenants in Identity and how to control the members who have access to them.
+
+### Managing tenants
+
+:::note
+The `<default>` tenant is automatically created during Identity startup. See here for more information.
```

Comment:
Did you mean to provide a link for **here**?

Reply:
Ah yes, I meant to actually add a comment here about this, I don't believe the related docs are in place yet (the conceptual side/migration side) so I can't link to anything just yet, should I leave this out?

Reply:
done!